main.html

```diff
@@ -83,154 +88,166 @@ <h1>
       </div>
     </section>
 
-  <!-------PROJECTS SECTION ------->
-
-  <section id="projects">
-    <div class="projects">
-      <h2 class="projects__title">
-        My Recent <span class="span__br"><br /></span>Works
-      </h2>
-      <div class="projects__h2-icon"></div>
-
-      <div class="projects__wrapper">
-        <article class="project__card card1">
-            <img src="./assets/producer1.png" alt="producer page demo" srcset="" />
-
-          <div class="project__card__description">
-            <h3 class="project__card__title">
-              Multi-Post Stories <br />
-              Gain+Glory
-            </h3>
-
-            <ul class="project__card__tags">
-              <li>Ruby on Rails</li>
-              <li>CSS</li>
-              <li>JavaScript</li>
-              <li>HTML</li>
-            </ul>
+ <!-------PROJECTS SECTION ------->
+
+ <section id="projects">
+  <div class="projects">
+    <h2 class="projects__title">
+      My Recent <span class="span__br"><br /></span>Works
+    </h2>
+    <div class="projects__h2-icon"></div>
+
+    <div class="projects__wrapper">
+      <article class="project__card card1">
+        <picture class="project__card__picture">
+          <img src="" alt="" srcset="" />
+        </picture>
+
+        <div class="project__card__description">
+          <h3 class="project__card__title">
+            Multi-Post Stories <br />
+            Gain+Glory
+          </h3>
+
+          <ul class="project__card__tags">
+            <li>Ruby on Rails</li>
+            <li>CSS</li>
+            <li>JavaScript</li>
+            <li>HTML</li>
+          </ul>
```

Comment:
@usorfaitheloho I'm not sure what to change because this is how I see my page on my desktop version 

<img width="1368" alt="Captura de pantalla 2023-05-11 a la(s) 15 32 04" src="https://github.com/rudicarrilloypr/portfolio/assets/125106965/da5b3782-65c2-49f3-9e7d-8510c55324a8">